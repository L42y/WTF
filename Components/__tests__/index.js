'use strict';

import React from 'react';
import renderer from 'react-test-renderer';

import App from '../';

test('<App> renders without user prop', () => {
  const tree = renderer.create(
    <App>test</App>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});

test('<App> renders with user prop', () => {
  const tree = renderer.create(
    <App user={{id: '123', username: 'L42y'}}>test</App>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
