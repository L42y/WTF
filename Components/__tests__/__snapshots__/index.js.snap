exports[`test <App> renders with user prop 1`] = `
<div
  className=" jsxstyle0"
  style={undefined}>
  <div
    className=" jsxstyle1"
    style={undefined}>
    <div
      className=" jsxstyle2"
      style={undefined}>
      <a
        onClick={[Function]}
        style={Object {}}>
        Frontend
      </a>
    </div>
    <div
      className=" jsxstyle3"
      style={undefined}>
      <div
        className=" jsxstyle4"
        style={undefined}>
        <div
          className=" jsxstyle5"
          style={undefined}>
          <a
            onClick={[Function]}
            style={Object {}}>
            Create
          </a>
        </div>
        <a
          onClick={[Function]}
          style={Object {}}>
          @
          L42y
        </a>
      </div>
    </div>
  </div>
  test
</div>
`;

exports[`test <App> renders without user prop 1`] = `
<div
  className=" jsxstyle0"
  style={undefined}>
  <div
    className=" jsxstyle1"
    style={undefined}>
    <div
      className=" jsxstyle2"
      style={undefined}>
      <a
        onClick={[Function]}
        style={Object {}}>
        Frontend
      </a>
    </div>
    <div
      className=" jsxstyle3"
      style={undefined}>
      <a
        onClick={[Function]}
        style={Object {}}>
        Login
      </a>
    </div>
  </div>
  test
</div>
`;
